lib/mountain_view/presenter.rb

```diff
@@ -0,0 +1,70 @@
+module MountainView
+  class Presenter
+    class_attribute :_attributes, instance_accessor: false
+    self._attributes = {}
+
+    attr_reader :slug, :properties
+    alias_method :props, :properties
+
+    def initialize(slug, properties={})
+      @slug = slug
+      @properties = attribute_defaults.deep_merge(properties)
+    end
+
+    def partial
+      "#{slug}/#{slug}"
+    end
```

Comment:
Oh, didn't know about that!
Yeah, overriding `partial_path` is definitively overkill. As is, `partial`, should be perfectly fine then.

Thanks for the research!